test/pdfs/.gitignore

```diff
@@ -419,3 +419,4 @@
 !issue12337.pdf
 !pr12564.pdf
 !pr12828.pdf
+!issue12716.pdf
```

Comment:
This should be a *linked* test-case, given that we don't have explicit permission to include it in the repository; please just link to https://github.com/mozilla/pdf.js/files/5670936/form-cms1500.pdf instead.

Reply:
Thank you for the hint. Didn't know this is possible.